features/a_product_should_have_a_category.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Product Categories
+  As a user
+  In order make it easier to find what I want
+  I would like to see a list of items ordered by category
+
+  Background: 
+    Given the following categories exists
+    | name      |
+    | Starter    |
+    | Main       |
+    | Dessert    |
+
+  
+    Given the following products exists    
+          | name         | description    | price | category    |
+          | Pizza1       | Cheesy         |   5   |  Main       |
+          | Pizza2       | Spicy          |   30  |  Main       |
+          | Cheese Stick | Spicy          |   30  |  Starter    |
+          | Tomato       | Spicy          |   30  |  Starter    |
+          | Cream        | Spicy          |   30  |  Dessert    |
+
+  Scenario: A product should have a category
+    Given I visit the landing page
+    Then I should see "5.00 kr" for "Pizza1"
+    And I should see "Spicy" for "Cheese Stick"
+    And I should see "30.00 kr" for "Cream"
+
```

Comment:
I used this PR to test the logic of the categories. Since another group was working on the product page from another branch at the same time (where we are really gonna make use of the categories and make them selectable), I decided to just implement one product per category on the landing page. The logic for this can be reused on the product page and once we have merged we will add another scenario for that. I guess I could make a step to test that only one product per category gets displayed but I felt it was redundant. Tell me if you want me to do that.

Reply:
Well it is a feature test about categories but no categories are actually tested here, so I just want you to test that at least one category, for now, is displayed. Otherwise this scenario is redundant.